DoLah/Parser/TaskTokenizer.cpp

```diff
@@ -0,0 +1,77 @@
+#include "TaskTokenizer.h"
+
+
+namespace DoLah {
+    std::vector<std::string> TaskTokenizer::DEADLINE_INDICATOR = { "on", "by" };
+    std::string TaskTokenizer::SCHEDULE_INDICATOR = "from";
+    std::vector<std::string> TaskTokenizer::SCHEDULE_SEPARATOR = { "to", "until" };
+    std::string TaskTokenizer::tag = "#";
+
+    TaskTokenizer::TaskTokenizer() {
+    }
+
+
+    TaskTokenizer::~TaskTokenizer() {
+    }
+
+    std::vector<std::string> TaskTokenizer::findTags(std::vector<std::string> lineArr) {
+        std::vector<std::string> tags;
+        std::vector<std::string> moretags;
```

Comment:
either provide necessary comment or improve the code, this is not so obvious to me.